src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 0.3s ease;
```

don't use all, it's not optimized in browser

```suggestion
    transition: color 0.3s ease, transform 0.3s ease;
```